operator/src/main/java/oracle/kubernetes/operator/helpers/DomainValidationSteps.java

```diff
@@ -105,12 +104,12 @@ public NextAction onSuccess(Packet packet, CallResponse<V1ConfigMapList> callRes
     }
 
     static List<V1ConfigMap> getConfigMaps(Packet packet) {
-      return Optional.ofNullable(getConfigMapsIfContinue(packet)).orElse(new ArrayList<>());
+      return Optional.ofNullable(getConfigMapsFromPacket(packet)).orElse(new ArrayList<>());
     }
 
     @SuppressWarnings("unchecked")
-    private static List<V1ConfigMap> getConfigMapsIfContinue(Packet packet) {
-      return packet.get(AsyncRequestStep.CONTINUE) != null ? (List<V1ConfigMap>) packet.get(CONFIGMAPS) : null;
+    private static List<V1ConfigMap> getConfigMapsFromPacket(Packet packet) {
+      return packet.get(CONFIGMAPS) != null ? (List<V1ConfigMap>) packet.get(CONFIGMAPS) : null;
```

Comment:
It looks like Dongbo's comment above is correct -- this code could be simplified or just inlined as it tests a key and then returns either the same value or null -- it could have just returned the packet.get() straight away.